packages/gatsby-theme-docs/src/layouts/internals/layout-page-header-side.js

```diff
@@ -4,16 +4,29 @@ import { designSystem } from '@commercetools-docs/ui-kit';
 const LayoutPageHeaderSide = styled.aside`
   grid-area: page-header-side;
   display: none;
+  padding: 0 ${designSystem.dimensions.spacings.m};
+  margin: ${designSystem.dimensions.spacings.m} 0
+    ${designSystem.dimensions.spacings.xl} 0;
```

Comment:
```suggestion
  margin: ${designSystem.dimensions.spacings.m} 0;
```

Reply:
But the spacing for margin-bottom should be bigger than for the top 🤔

Reply:
That spacing you're talking about (I suppose) is something else and depends on the page heading.
For example

<img width="1117" alt="image" src="https://user-images.githubusercontent.com/1110551/100110696-902b4580-2e6d-11eb-8165-c73460d215dd.png">

The margin here should be consistent top/bottom, just for safety measures.

Reply:
Allright 👍